tests/unit_tests/__init__.py

```diff
@@ -0,0 +1,50 @@
+import unittest
```

Comment:
I am not sure, if this file could better remain empty and all the useful stuff from here goes to a file e.g. called `base_test.py` in analogy to what we did in `service-stac`?

Reply:
Ok, should we leave `__init__.py` file always empty? (python noob question)

Reply:
Probably there is not really a right or wrong in this case.
It just would make things a bit more clear, to have a dedicated module for this rather than putting it in the `__init__.py`. 

There are different ways to use the `__init__.py` and certain use cases, where a non-empty `__init__.py` is useful:
https://pcarleton.com/2016/09/06/python-init/